CHANGELOG.md

```diff
@@ -2,6 +2,13 @@ UNRELEASED
 ----------
 
 
+
+2022.4.13
```

Comment:
Please do not add the version number, just list your issue under the "UNRELEASED" section.
The version number will automatically be updated when the new version is released.
Also, please rebase your branch to resolve the conflicts in the changelog. Thanks!